pkgs/by-name/db/dbt-duckdb/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+}:
+  python3Packages.buildPythonApplication rec {
+    pname = "dbt-duckdb";
+    version = "1.9.6";
+    pyproject = true;
+
+  dependencies = with python3Packages; [
+    pbr
+  ];
+
+  src = fetchFromGitHub {
+    owner = "duckdb";
+    repo = "dbt-duckdb";
+    tag = version;
+    hash = "sha256-zuSOM5Vfr89aVGDcVQtK1Y4c3hRcL8pRscnvpeP6jbU=";
+  };
+
+  build-system = [
+    python3Packages.setuptools
+    python3Packages.wheel
+  ];
+
```

Also 
```suggestion
  env.PBR_VERSION = version;
```
is important, because PBR doesn't like Github tarballs